### PR TITLE
Update Dockerfile and add .dockerignore

### PR DESCRIPTION
**Pull request by fwendland**
-  Add labels
-  Add Codyze binary to PATH
-  Update entrypoint and supply default command parameters
-  Add comments to Dockerfile
-  Add only built distributions to Docker build context via .dockerignore

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,23 @@
 FROM openjdk:11-jre-slim
 
+LABEL org.opencontainers.image.authors="Fraunhofer AISEC <codyze@aisec.fraunhofer.de>" \
+      org.opencontainers.image.licenses="Apache-2.0" \
+      org.opencontainers.image.source="https://github.com/Fraunhofer-AISEC/codyze" \
+      org.opencontainers.image.vendor="Fraunhofer AISEC" \
+      org.opencontainers.image.base.name="openjdk:11-jre-slim"
```
Comment (oxisto) [inline Dockerfile:7]: Does it make sense to have this as a label? Seems redundant

Comment (fwendland) [reply Dockerfile:7]: While we host on GitHub, this information is readily available. In this sense it's redundant. Do we intend to distribute Codyze container images somewhere else?

Comment (oxisto) [reply Dockerfile:7]: > While we host on GitHub, this information is readily available. In this sense it's redundant. Do we intend to distribute Codyze container images somewhere else?

For now, its only planned to host it on GH.

Comment (fwendland) [reply Dockerfile:7]: Then I'll remove it.

**Review by oxisto — APPROVED** [review]

**Pull request merged**